kiwi/container/docker.py

```diff
@@ -113,12 +116,24 @@ def create(self, filename):
             [container_dir, self.container_tag]
         )
 
-        Command.run(
-            ['umoci', 'init', '--layout', container_dir]
-        )
-        Command.run(
-            ['umoci', 'new', '--image', container_name]
-        )
+        if base_image:
+            uncompressor = Compress(base_image)
+            uncompressor.uncompress(True)
+            self.uncompressed_image = uncompressor.uncompressed_filename
+
+            Command.run([
+                'skopeo', 'copy',
+                'docker-archive:{0}'.format(self.uncompressed_image),
+                'oci:{0}'.format(container_name)
+            ])
```

Comment:
Actually this was the mentioned drawback of my proposal in the trello card, having to do such a strange movement, I am in favor of finding a nicer solution, but no ideas so far.